salt/_states/metalk8s.py

```diff
@@ -76,6 +76,7 @@ def static_pod_managed(name,
             context or {},
             config_digest=config_digest, metalk8s_version=metalk8s_version
         ),
+        tmp_dir=kwargs.pop("tmp_dir", "/tmp"),
```

Comment:
Instead of hardcoding `/tmp`, you can probably use [`tempfile.gettempdir`](https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir) which would be more reliable.

Reply:
Done.